src/js/components/StatusBar.js

```diff
@@ -7,7 +7,7 @@ const GROW_RATIO = 0.07;
 // Grow anything below this threshold by stealing from the biggest portion
 const MIN_SIZE = 7; // (% of statusBar width)
 
-function stealPortion(barSizes, indexesLessThanThreshold) {
+function stealPortion(barSizes, indexesLessThanThreshold, unassignedPortion) {
   if (indexesLessThanThreshold.length) {
```

Comment:
If we do like so, like we do in other places, we can unindent the whole function:
```js
if (!indexesLessThanThreshold.length) {
  return barSizes;
}
```